css/main.css

```diff
@@ -43,23 +43,23 @@ h1 {
   text-align: center;
 }
 
-.text {
+.exp-the-rush-section__desc {
   font-size: 20px;
   font-family: 'Lato', sans-serif;
   color: rgb(168, 123, 183);
   text-align: center;
   line-height: 1.65;
-  margin-top: 6px;
+  padding-top: 6px;

```

Comment:
this is not a BEM-related change

Reply:
This change is for `.exp-the-rush-section__desc` class. This class is assigned for Element tag. According to BEM Element can't set position of itself. So this change has indirect relation to BEM.